supertokens/tenantConfigMapping.go

```diff
@@ -0,0 +1,90 @@
+package supertokens
+
+import (
+	"errors"
+)
+
```

Comment:
all these need to go in the recipe interface / impl

Reply:
Done